src/Sql/Sql.Test/ScenarioTests/DatabaseBackupTests.cs

```diff
@@ -38,7 +38,7 @@ public void TestListDatabaseRestorePoints()
             RunPowerShellTest("Test-ListDatabaseRestorePoints");
         }
 
-        [Fact]
+        [Fact(Skip = "Not recordable")]
```

Comment:
We need to ge a positive confirmation from the test owner (or other knowledgeable dev) that this behavior has not regressed.

Reply:
> We need to ge a positive confirmation from the test owner (or other knowledgeable dev) that this behavior has not regressed.

Signed off DatabaseBackuTests.cs and ps1